### ISSUE TITLE
Upgrade to AGP 7.1.1

### ISSUE THREAD

**Issue by AbdelrahmanTalat** [cashapp/paparazzi#372]
Hi there I am having an issue compiling my project with the latest version of paparazzi I believe the issue is related to this [bug](https://issuetracker.google.com/issues/189559355?pli=1) and it has been resolved by consequent versions. 

`A problem occurred configuring project ':ui-module'.
> Failed to notify project evaluation listener.
   > java.lang.IllegalArgumentException: Cannot change attributes of dependency configuration ':ui-module:releaseUnitTestCompileClasspath' after it has been resolved
   > KotlinJvmAndroidCompilation with name 'releaseUnitTest' not found.`

**Comment by jrodbx** [comment]
The referenced bug was fixed in Android Gradle Plugin 7.0.0-beta04, which the recent bump to AGP 7.0 should have fixed.

**Comment by eboudrant** [comment]
@jrodbx I experience the same issue and we're using AGP 7.1.1.

Here the error : 
```
* What went wrong:
A problem occurred configuring project ':module'.
> Failed to notify project evaluation listener.
   > java.lang.IllegalArgumentException: Cannot change attributes of dependency configuration ':module:releaseUnitTestCompileClasspath' after it has been resolved
   > KotlinJvmAndroidCompilation with name 'releaseUnitTest' not found.
```
According the bug that should be fixed in 7.1.1 so that's unexpected, were you able to use paparazzi with AGP 7.1.1?

**Comment by jrodbx** [comment]
Yes, and we're now using 7.1.2 on CashApp.  Possibly an issue with another plugin?
